Formula/tidy-viewer.rb

```diff
@@ -0,0 +1,19 @@
+class TidyViewer < Formula
+  desc "CLI csv pretty printer"
+  homepage "https://github.com/alexhallam/tv"
+  url "https://github.com/alexhallam/tv/archive/refs/tags/1.4.3.tar.gz"
+  sha256 "e308eb088d059d18119dc757c98487d9cabc2f4b97035a8dc9f8253717aa9fe9"
+  license "Unlicense"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+    bin.install_symlink "tidy-viewer" => "tv"
+  end
+
+  test do
+    (testpath/"test.csv").write("first header,second header")
+    system bin/"tv", "test.csv"
```

Comment:
Can we check some output here too?

Reply:
Sorry, I'm not really familiar with ruby. Do you have an example by any chance?

Reply:
Something like this should work
```suggestion
    assert_match "first header", shell_output("#{bin}/tv", "test.csv")
```

Reply:
@SMillerDev I did not manage to add what you recommended, because the output of this tool is a bit complex:
![image](https://user-images.githubusercontent.com/714056/150756551-0755c7a7-292e-440f-9dcd-fdb172de280c.png)

Is there any way to merge as is, or get some help to achieve this?

Reply:
That would work with the command I suggested.

Reply:
> That would work with the command I suggested.

Unfortunately no: https://github.com/Homebrew/homebrew-core/actions/runs/1739314985